resources/views/home/index.blade.php

```diff
@@ -49,6 +49,24 @@ class="tw-relative tw-mt-3 tw-inline-block tw-border tw-border-gray-300 tw-px-4
                 </div>
             </div>
         @endif
+
+        {{-- Counts blurb --}}
+        <div class="tw-bg-gray-200">
+            <div
+                class="tw-container tw-mx-auto tw-grid tw-max-w-screen-xl tw-px-6 tw-py-5 tw-text-gray-500 lg:tw-py-10">
+                <p class="tw-text-center lg:tw-text-2xl">
+                    {{ utrans('intro.definition_start') }}
+                    <a href="{{ route('frontend.catalog.index') }}"
+                        class="tw-font-bold tw-text-gray-800 hover:tw-underline">{!! formatNum($countsBlurb->itemsCount) !!}</a>
+                    {{ trans('intro.definition_end') }}<br>
+                    {{ $countsBlurb->parts[0] }}
```

Comment:
from my point of view - this is not very intuitive. to use "part + array index". at least I would prefer to make it "associated array" - to use string instead of number. something like
```suggestion
                    {{ trans($countsBlurb->parts['claim_start']) }}
```